### PR TITLE
Player fixes

### PR DESCRIPTION
**Pull request by garbear**
Here's a collection of random improvements to the video and audio players I've done over the past month.

The `GUIInfoManager` and `SeekHandler` commits deal with casting time amounts into ints. The commits replace an int with a double and use proper rounding for casts of time amounts.

### CODE REVIEW

File: xbmc/GUIInfoManager.cpp [hunk 1]
```diff
@@ -6478,16 +6478,16 @@ bool CGUIInfoManager::GetInt(int &value, int info, int contextWindow, const CGUI
             {
               const CEpgInfoTagPtr tag(GetEpgInfoTag());
               if (tag)
-                value = static_cast<int>(tag->ProgressPercentage());
+                value = MathUtils::round_int(tag->ProgressPercentage());
```
Comment (tamland) [inline xbmc/GUIInfoManager.cpp]: std::round?

Comment (garbear) [inline xbmc/GUIInfoManager.cpp]: didn't know this was added in c++11. will change to std::lround()

**Comment by FernetMenta** [comment]
+1 looks good as far as I can tell

**Comment by garbear** [comment]
Updated MathUtils::round_int() to std::roundl(). jenkins build this please

**Comment by garbear** [comment]
oops, std::lround() instead of std::roundl(). jenkins build this please

**Comment by garbear** [comment]
hm, android (exclusively) is failing with 'std::lround() not found'. Any ideas?

**Comment by hudokkow** [comment]
I assume Android doesn't fully support C++11 so std::lround() is missing.
For instance, std::to_string() is missing, so is std::stoi(), etc.

**Comment by garbear** [comment]
k, I'll change back to MathUtils::round_int() then

**Comment by garbear** [comment]
jenkins build this please

**Comment by ksooo** [comment]
> I assume Android doesn't fully support C++11 so std::lround() is missing.

 Yep, unfortunately. Stumbled myself over exactly this some time ago.

**Pull request merged**